custom_components/opnsense/coordinator.py

```
@@ -119,144 +106,146 @@
+            {
+                "function": "get_unbound_blocklist",
+                "state_key": ATTR_UNBOUND_BLOCKLIST,
+            },
+        ]
+
+        self._state.update(await self._get_states(categories))
+
+        # self._state["dhcp_leases"] = []
+        self._state["dhcp_stats"] = {}
+        self._state["dhcp_stats"]["leases"] = await self._get_dhcp_stats(
+            self._state.get("dhcp_leases", [])
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)